Data/Models/Academies/LatestOfstedJudgement.cs

```diff
@@ -7,26 +8,17 @@ public class LatestOfstedJudgement
         public string SchoolName { get; set; }
         public string InspectionDate { get; set; }
         public string OverallEffectiveness { get; set; }
-        public string AchievementOfPupils { get; set; }
-        public string QualityOfTeaching { get; set; }
-        public string BehaviourAndSafetyOfPupils { get; set; }
-        public string LeadershipAndManagement { get; set; }
-        public string EarlyYearsProvision { get; set; }
-        public string SixthFormProvision { get; set; }
+        
+        [DisplayName("Ofsted Report")]
+        public string OfstedReport { get; set; }
 
         public IEnumerable<FormField> FieldsToDisplay()
```

Comment:
We can probably remove this method tbh and make the view call the methods it depends on directly. It was a hangover from an idea I had a while ago but I dont think it really holds up now.

If you remove it, remove it from the preview page too. That entire page is gonna have to be re-done so we can just delete that.